renderdoc/driver/gl/gl_driver.cpp

```diff
@@ -2878,6 +2878,7 @@ WrappedOpenGL::BackbufferImage *WrappedOpenGL::SaveBackbufferImage()
 
   byte *jpgbuf = NULL;
   int len = thwidth * thheight;
+  len = len >= 1024 ? len : 1024;
```

Comment:
Yeah, this happened when one of the texture failed to copy for some reason and the size defaulted to an 1x1 texture.